glean-core/src/database/mod.rs

```diff
@@ -147,6 +165,59 @@ impl Database {
         }
     }
 
+    /// Checks if the 'user' lifetime metric is expired and, if so, removes it.
+    ///
+    /// ## Arguments
+    ///
+    /// * `storage_name`: The storage name to iterate over.
+    /// * `metric_id_slice`: The metric id to check.
+    /// ## Return value
```

Comment:
Newline before this.
```suggestion

    /// ## Return value
```